integration/token/interop/views/accept.go

```diff
@@ -19,13 +22,29 @@ func (a *AcceptCashView) Call(context view.Context) (interface{}, error) {
 	// to a request for a recipient identity.
 	// The recipient can do that by using the following code.
 	// The recipient identity will be taken from the default wallet (ttx.MyWallet(context)), if not otherwise specified.
-	_, err := ttx.RespondRequestRecipientIdentity(context)
+	me, err := ttx.RespondRequestRecipientIdentity(context)
 	assert.NoError(err, "failed to respond to identity request")
 
 	// At some point, the recipient receives the token transaction that in the meantime has been assembled
 	tx, err := ttx.ReceiveTransaction(context)
 	assert.NoError(err, "failed to receive tokens")
 
+	outputs, err := tx.Outputs()
+	assert.NoError(err, "failed to get transaction's outputs")
+	now := time.Now()
+	for i := 0; i < outputs.Count(); i++ {
+		output, err := htlc.ToOutput(outputs.At(i))
+		assert.NoError(err, "cannot get htlc output wrapper")
+		if !output.IsHTLC() {
+			continue
+		}
+		// check script details, for example make sure the deadline has not passed
```

Comment:
`AcceptCashView` is only used to accept when there is an issue (according to latest topology) and so this code is never used.

Reply:
aha, good catch. It is during Lock. We have already stuff there. I will update.